.github/workflows/create-release.yml

```diff
@@ -65,4 +76,4 @@ jobs:
         target_commitish: ${{ github.sha }}
         name: v${{ steps.tag.outputs.tag }}
         body: "See builder.toml file"
-        draft: true
```

Comment:
Thought I'd share something I noticed:
I think the call to "reset-draft" action is redundant. It won't output anything when there are no draft releases present. If the tag action is called without an input var, it should automatically +1 the patch segment of the tag for you. Not sure if you decided to keep the "reset-draft" on purpose.

This looks like an example of doing the exact reverse of what you're doing here: https://github.com/paketo-buildpacks/github-config/commit/00af31bcef9fb89fe3112d5afabde91da0407295

Reply:
@arjun024 My thought was that by leaving it, we'd be able to manually set the version of the subsequent release if desired (i.e. to comply with some kind of semantic versioning in the future). Does that make sense? It's definitely a no-op for now though. Happy to remove for simplicity.

Reply:
My inkling was right - you left it for a greater purpose. Sounds good to me